crates/sui-framework/packages/sui-framework/sources/bridge/bridge.move

```diff
@@ -0,0 +1,363 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+#[allow(unused_const)]
+module sui::bridge {
+    use std::vector;
+
+    use sui::address;
+    use sui::balance;
+    use sui::bcs;
+    use sui::bridge_committee::{Self, BridgeCommittee};
+    use sui::bridge_escrow::{Self, BridgeEscrow};
+    use sui::bridge_treasury::{Self, BridgeTreasury, token_id};
+    use sui::chain_ids;
+    use sui::coin::{Self, Coin};
+    use sui::event::emit;
+    use sui::object::{Self, UID};
+    use sui::table::{Self, Table};
+    use sui::transfer;
+    use sui::tx_context::{Self, TxContext};
+
+    #[test_only]
+    use sui::bridge_treasury::USDC;
+    #[test_only]
+    use sui::hex;
+    #[test_only]
+    use sui::test_scenario;
+
+    struct Bridge has key {
+        id: UID,
+        // nonce for replay protection
+        sequence_num: u64,
+        // committee pub keys
+        committee: BridgeCommittee,
+        // Escrow for storing native tokens
+        escrow: BridgeEscrow,
+        // Bridge treasury for mint/burn bridged tokens
+        treasury: BridgeTreasury,
+        pending_messages: Table<BridgeMessageKey, BridgeMessage>,
```

Comment:
`Table` doesn't allow you to iterate through all the entries. It might make sense to use `LinkedTable` instead which gives you this ability, in case it becomes useful later on.